QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
 QUEST_20150317_000045	Select GenType
 QUEST_20150317_000046	Select Coordinates
 QUEST_20150317_000047	Knight Titas
-QUEST_20150317_000048	If that's correct, go and see Knight Commander Uska, at Klaipeda. {nl}But first, I must speak with you.
-QUEST_20150317_000049	It would be nice if I had a moment to relax…
+QUEST_20150317_000048	If that is so then go see Knight Commander Uska at Klaipeda. {nl}But first, I must speak with you. Follow me.
+QUEST_20150317_000049	It would be nice if I had a moment to rest…
 QUEST_20150317_000050	Sparnas… {nl}Not someone to take lightly…
 QUEST_20150317_000051	Soldier Moody
 QUEST_20150317_000052	Let's deal with this monster before talking.
 QUEST_20150317_000053	Joint Investigation Team - Scott
-QUEST_20150317_000054	I long for a time when our team is no longer needed… {nl}Ah, that'll be a peaceful age.
+QUEST_20150317_000054	I long for a time where our team is no longer needed… {nl}What a peaceful age that would be…
```

Comment:
Changing "when" to "where" changes the meaning a bit.  Instead of longing for a time in the future, hopeful that this happens, it begins to sound more like they are looking for a time ( like a location "where", in general ).  I believe the original line is just fine as is.